### ISSUE TITLE
ZippedMoabVersion count less than number of object versions per PreservedObject

### ISSUE THREAD

**Issue by ndushay** [sul-dlss/preservation_catalog#2029]
from dashboard: 

![image](https://user-images.githubusercontent.com/96775/200701119-d68f3136-ec1a-4cc5-876a-147db3a7fe30.png)

blocked by #2026, #2028 - once those errors are addressed, we will want to address the replicated count mismatch.

It may be a matter of 
- running Catalog to Archive
- tweaking how Catalog to Archive works to find the missing ZMVs

We may want code In order to find the missing ZMVs that can be run as a rake task or via the UI.

Then we will want a way to fix the discrepancies. and then create them and replicate their ZipParts, not unlike #2026, #2028

**Comment by ndushay** [comment]
Late breaking news:  existing CatalogToArchive code might be able to fix these if backfill replication feature is turned on:   https://github.com/sul-dlss/preservation_catalog/issues/2036

**Comment by jmartin-sul** [comment]
i believe that missing ZippedMoabVersions indicate versions that have not been replicated to endpoints.  as we start cleaning up database records and feeding lists of partially replicated druid versions to ops, a missing ZippedMoabVersion _may_ indicate a database record that's been cleaned up with a possibly lingering partial copy in S3 that hasn't been cleaned up (see #2401).

but since the `CatalogToArchive` audit runs over `PreesrvedObject`s and their child `ZippedMoabVersion`s, but does not create `ZippedMoabVersion`s (unless we turn on back filling see #2036), and since it doesn't scour all _possible_ S3 bucket locations for things that might've been replicated but aren't recorded (since we don't know of a way for that to happen), I don't think it is possible for "runing Catalog To Archive" to "find the missing ZMVs" (because AFAIK, we should treat any cloud copy that doesn't have a database record as suspect).

I sort of suspect that this ticket is mostly duplicative of different aspects of #2036 and #2041 and #1733.  But I also like the idea of tracking this remediation issue as its own high level task, because of the way the dashboard exposes it. So I'm hesitant to close it even if the work to do it is already tracked in other tickets.

**Comment by ndushay** [comment]
In theory, Ed is supposed to do something to address these errors per ticket #2081.  I don't think this is ready until we know what doesn't get addressed by that.

**Comment by mjgiarlo** [comment]
OK, thanks! Marked as blocked by #2081

**Comment by ndushay** [comment]
haha - #2145 shows ZippedMoabVersion count greater than the number of versions per PreservedObject.

**Comment by mjgiarlo** [comment]
Super!
